OutSystems/outsystems-datagrid-reactive
# ROU-3054: Style client actions should return error message and success boolean

This PR adds feedback messages to Style API methods. All methods did not return any feedback on whether or not the operation was successful or if any errors happened. Also, fixes an [issue reported on this forum post](https://www.outsystems.com/forums/discussion/73148/add-and-remove-css-classes/)

### Test Steps

1. Click in "AddCellCssClass":
Expected: The first cell from ProductName should become red (you might need to click on another cell) and a success message should appear.
2. Click in "RemoveAllCssClassesFromCell":
Expected: The first cell from ProductName should have the default color and a success message should appear
 
3. Click in "AddColumnCssClass":
Expected: The ProductName column should become red and a success message should appear.
4. Click in "RemoveCssClassesFromColumn":
Expected: The ProductName column should have the default color and a success message should appear.

5. Click in "AddClass":
Expected: The first row column should become red and a success message should appear.
6. Click in "RemoveClass":
Expected: The first row should have the default color and a success message should appear.

7. Click in "AddClass":
8. Click in "Remove All Classes From Row ":
Expected: The first row should have the default color and a success message should appear.


1. Click in "AddCellCssClass invalid":
Expected: Error message should show saying: "Error"

2. Click in "RemoveAllCssClassesFromCell  invalid":
Expected: Error message should show saying: "Error"
 
3. Click in "AddColumnCssClass  invalid":
Expected: Error message should show saying: "Error"

4. Click in "RemoveCssClassesFromColumn  invalid":
Expected: Error message should show saying: "Error"

5. Click in "AddClass invalid" :
Expected: Error message should show saying: "Error"

6. Click in "RemoveClass  invalid":
Expected: Error message should show saying: "Error"

7. Click in "Remove All Classes From Row  invalid":
Expected: Error message should show saying: "Error"
